src/panels/config/automation/condition/ha-automation-condition-row.ts

```diff
@@ -239,23 +262,26 @@ export default class HaAutomationConditionRow extends LitElement {
   private async _handleAction(ev: CustomEvent<ActionDetail>) {
     switch (ev.detail.index) {
       case 0:
-        await this._renameCondition();
+        await this._testCondition2();
```

Comment:
2?

Reply:
LOl yeah, let me rename that. Copy the old one to test them next to each other.